### PR TITLE
Done with javascript xmlhttprequest and api calling

### PR DESCRIPTION
**Pull request by allurivamsi5**

### CODE REVIEW

File: templates/login.html [hunk 1]
```diff
@@ -2,18 +2,60 @@
...
+                htmltext= htmltext + "<div><div class='card ' style='width: 40rem; background-color: bisque;'>"+
+                        "<div class='card-body'><h5 class='card-title' style = 'color:black'>Email:"+data.reviews[i]["email"]+"</h5><h6 class='card-subtitle mb-2 text-muted' style = 'color:black'>Rating: "+data.reviews[i]["rating"]+
+                        "</h6><p class='card-text' style = 'color:black'>Comment: "+data.reviews[i]["comment"]+"</p></div></div></div>"  
+            }
+            htmltext = htmltext 
+            document.getElementById("reviewresults").innerHTML=htmltext;
+
+	  } else {
+		  // console.error("Fail");
+      if (xhr.status == 404) {
```
Comment (ashishk144) [inline templates/login.html:42]: Use the error message from the API itself.

File: templates/login.html [hunk 2]
```diff
@@ -2,18 +2,60 @@
...
 </style>
 {% endblock %}
+{% block scripts%}
+  <script type="text/javascript"> 
+  function bookdisplay(ele) {
+    console.log(ele.id)
+    var requestobj = {}
+    requestobj["isbn"] = ele.id
+    var url = "/api/book?isbn="+ele.id
+    // document.getElementById("id").innerHTML=requestobj.isbn;
```
Comment (ashishk144) [inline templates/login.html]: remove all the commented pieces of code.

File: templates/login.html (hunk 1 above)
Comment (allurivamsi5) [reply templates/login.html:42]: updated, bro..

File: templates/login.html (hunk 2 above)
Comment (allurivamsi5) [reply templates/login.html]: updated..

**Pull request merged**